packages/cf-style-const/src/variables.js

```diff
@@ -97,5 +97,5 @@ export default {
   colorFacebookBlue: '#3b5998',
   colorMainBackground: '#e6e6e6',
   zIndexMax: 1000,
-  modalBoxShadow: '0 0 0 9999px rgba(0, 0, 0, .7)'
+  modalBackdropBackground: 'rgba(0, 0, 0, .7)'
```

Comment:
> Maybe something like overlayColor: 'rgba(0,0,0,0.7)' ?

That would be very reasonable to me. If you want this to be in sync with the existing modals in the CF Dash (not sure whether those are "react" or "bb" with @manatarms  terminology) though, you’d want to find out where the existing `background: #000` and `opacity: .7` are coming from.